plugins/amazonq/codewhisperer/jetbrains-community/src/Hello.kt

```diff
@@ -0,0 +1,15 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.ui.MessageDialogBuilder
+import software.aws.toolkits.resources.AmazonQBundle
+
+class Hello : AnAction(AmazonQBundle.message("q.hello")), DumbAware {
```

Comment:
Test class, can be deleted?

Reply:
whoever moves the code can delete it.
it's a sample to show what we're expecting